AirSkyBoat/AirSkyBoat
# Dynamis Tav TP Move Audit

**_I affirm:_**


Tested what is currently done - everything that is coded works as expected.

## Please enter a player-facing description

Audited Dynamis Tav TP Moves using multiple sources:
- Skinoz's amazing captures (in capture disco)
- Random YouTube Videos like https://youtu.be/2dM2nGm3Jf8?t=42 (Heavy Bellow is full AOE)
- https://www.dynamisbums.com/strategy/tav.html (Lots of mostly accurate data)
- https://www.bluegartr.com/threads/90841-Nightmare-Worm-Spirit-Vacuum
- Rolled back wikis: https://ffxiclopedia.fandom.com/wiki/Nightmare_Antlion
- http://www.dynamisbums.com/forum/showthread.php?1650-Dynamis-Tavnazia-(Quick-guide-to-winning)
- https://www.bluegartr.com/threads/41453
- https://enedin.be/dyna/html/zone/frame_tav.htm

Setting mob size correctly per in era videos
- Bugard, worm, and antlion are large sized
- Main nightmare mobs are large sized, with their spawns on aggro being medium sized
- There is no clear understanding of if the size means anything.  Could buff the larger ones to be tougher

Increased damage on PitAmbush to 3x.
Increased damage on Dyna Tav Pit Ambush to much more.

Added damage to Gastric Bomb.

Put in a temp fix to ensure clusters have the correct animationsubs - all cluster TP moves are based on mob animation sub.
This is why clusters in ro'maeve that are DEs dont use most of their TP moves, or the ones they use hit like a wet paper towel.

Updated delay for all nightmare mobs

Added Starlight Music on Umbral Aggro

**ToDo**
Overall balancing for damage in/out and drops.
Diabolos x 4 bosses TP moves, spells, and fight scripts

## What does this pull request do? (Please be technical)

Re-introduces mob skills with IDs which are uniquely used in Dyna Tav
Changes Mob skill lists to use new skills
Updates Pools to use skill lists
Updates modfied skill luas relying on the unique mobskill IDs

sets flags on mob pools for sizing
increased HP even more for the singular NMs
sets flags dynamically for Nightmare mobs for sizing
hacks a fix in for cluster animationsub
Implemented boss spawning and superlinking
Bypassed mob mnk damage correct for Nightmare Taurus.

## Steps to test these changes
Run Dyna~

Important skill differences:

Bugard:
- Heavy Bellow is AOE stun - not conal (see link above)
- Awful eye adds petrify
- Tusk has large knockback
- Scutum is 2x defense boost

Worm
- New move spirit vacum that absorbs 87% of tp
- Gastric bonb is 90% attack down
- tremors has higher dmg
- Sandspin is -200 acc

Antlion
- Pit Ambush should hurt alot from reports - putting a pld in critical or outright killing
- Venom Spray - 50/tick poison
- Sand Blast - AoE blind and silence

Tauri
- Backswish - larger knockback
- Mow - 24/tick poison - works if tauri are lvl 84

Leech
- Acid mist is 99% reduction
- Sand Breath  - -80acc

Makara
- Water Wall - 3x defense
- Water Shield - +50 evasion
- Intimidate - 66.66% slow - overwtires haste

Clusters
 - Needs testing.  
 - Self Destruct is very strong in retail current day captures
 - Ive heard personal reccolection that sling bomb should be feared
 - Ive heard personal reccolection that circle of flames was bad enough that they used to mana burn clusters only.

## Special Deployment Considerations

All the SQL !
